app/controllers/api/v1/ping_things_controller.rb

```diff
@@ -11,22 +11,30 @@ class InvalidRecordCount < StandardError; end
       def index
         limit = [(index_params[:limit] || 240).to_i, 9999].min
         page = index_params[:page] || 1
-        with_total_count = index_params[:with_total_count].to_s == "true" ? true : false
+        with_stats = index_params[:with_stats].to_s == "true" ? true : false
 
         ping_things = PingThing.where(network: index_params[:network])
                                .includes(:user)
                                .order(reported_at: :desc)
                                .page(page)
                                .per(limit)
 
-        total_count = PingThing.where(network: index_params[:network]).count if with_total_count
-
         json_result = ping_things.map { |pt| create_json_result(pt) }
 
-        if with_total_count
+        if with_stats
+          total_count = PingThing.where(network: index_params[:network]).count
+          p90 = PingThing.where(network: index_params[:network], reported_at: (DateTime.now - 5.minutes)..DateTime.now)
```

Comment:
Missing (
should be:
`reported_at: ((DateTime.now - 5.minutes)..DateTime.now)`